Application/Requests/MarketRequests/CreateMarketCommad.cs

```diff
@@ -5,13 +5,12 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Application.Markets.Commands
+namespace Application.Requests.MarketRequests
 {
-    public class CreateMarketCommand : IRequest<string>
+    public class CreateMarketCommand : IRequest<int>
     {
         public string MarketName { get; set; }
```

Comment:
make it as just Name, code etc

Reply:
`using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Domain.Enums;
using Domain.Enums.Domain.Enums;

namespace Domain.Model
{
    public class Market : AuditableEntity
    {
        public int Id { get; set; }  // Primary Key

        [Required]
        [MaxLength(150)]
        public string Name { get; set; }  // Unique

        [Required]
        [MaxLength(2)]
        public string Code { get; set; }  // Unique, 2 letters

        [Required]
        [MaxLength(50)]
        public string LongMarketCode { get; set; }  // Optional field for long code

        [Required]
        public Region Region { get; set; }  // Enum for Region

        [Required]
        public SubRegion SubRegion { get; set; }  // Enum for SubRegion

        // Navigation property for related MarketSubGroups
        public List<MarketSubGroup> MarketSubGroups { get; set; } = new List<MarketSubGroup>();
    }
}
`